src/main/java/com/google/sps/servlets/utils/CommentAnalysis.java

```diff
@@ -0,0 +1,69 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.servlets.utils;
+
+import com.google.api.services.youtube.model.CommentThread;
+import com.google.api.services.youtube.model.CommentThreadListResponse;
+import com.google.cloud.language.v1.Document;
+import com.google.cloud.language.v1.LanguageServiceClient;
+import com.google.cloud.language.v1.Sentiment;
+import java.io.IOException;
+import java.util.List;
+import java.util.stream.Collectors;
+
+/**
+ * This class encapsulates each element in json comment array into separate user comment object
+ * and does sentiment analysis on each of them.
+ */
+public class CommentAnalysis {
+  private LanguageServiceClient languageService;
+
+  public CommentAnalysis() throws IOException {
+    this.languageService = LanguageServiceClient.create();
+  }
+
+  /**
+   * It computes an overall statistics score from the retrieved youtube comments.
+   * @return a Statistics object that contains required values to display
+   */
+  public Statistics computeOverallStats(CommentThreadListResponse youtubeResponse) {
+
+    List<Double> scoreValues = youtubeResponse.getItems().stream()
+                                   .map(UserComment::new).map(this::calcualateSentiAnalysisScore)
```

Comment:
This is a nit that we never talked about before I guess. When we call a change of methods like here, we want to align the dots. So it would be:
```
youtubeResponse.getItems()
               .stream()
               .map(UserComment::new)
               .map(this::calcualateSentiAnalysisScore)
               .collect(Collectors.toList());
```
(took me a while to format it here :P)

Reply:
Please update all the files accordingly.

Reply:
Updated